dist/components/GlobalContext.d.ts

```diff
@@ -0,0 +1,6 @@
+import React from 'react';

```

Comment:
we should not be checking in the dist folder. is there a specific reason for adding it